gateway-js/src/make-fetch-happen.d.ts

```diff
@@ -33,6 +33,10 @@ declare module 'make-fetch-happen' {
           randomize?: boolean;
         };
     onRetry?(): void;
+    cert?: string;
+    key?: string;
+    proxy?: string;
```

Comment:
```suggestion
    proxy?: string | URL;
```

We'll also need to add the proper import at the top:
`import { URL } from 'url';`

Reply:
Ref: https://github.com/npm/make-fetch-happen#opts-proxy